modules/ROOT/pages/deploy-maven-4.x.adoc

```diff
@@ -3,7 +3,7 @@
 Runtime Fabric supports using Maven to manage and deploy Mule applications using Maven. The information in this topic is specific to deploying an application to Mule runtime, version 4.
 
 [NOTE]
-To deploy an application to Runtime Fabric, the application must be published in Exchange. You can use Studio, as describe in  xref:studio::export-to-exchange-task.adoc[Publishing a Project to Exchange]. Alternatively, use the Mule Maven Plugin. See xref:exchange::to-publish-assets-maven.adoc[Publish and Deploy Exchange Assets Using Maven] for more information.
+To deploy an application to Runtime Fabric, the application must be published in Exchange. You can use Studio, as described in  xref:studio::export-to-exchange-task.adoc[Publishing a Project to Exchange]. See xref:exchange::to-publish-assets-maven.adoc[Publish and Deploy Exchange Assets Using Maven] for more information.
```

Comment:
Instead of using a Note, I think this text can just follow the opening sentence, be a next paragraph, if you want it to be more distinct.

Reply:
...published in Exchange > published in Anypoint Exchange (Exchange). You can use Anypoint Studio (Studio), as described....

QUESTION: You can use Studio to do what? Publish the app in Exchange? Or just use Studio WITH an Exchange-published app deployed to Runtime Fabric? Clarify, unless I am just way more unfamiliar with all this than your audience...which can be an issue....  ;-)

Reply:
Removed the note and added clarification - thank you!